packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -95,6 +95,7 @@ export class Breadcrumb extends BaseComponent<IBreadcrumbProps, any> {
                   iconProps={ { iconName: 'More' } }
                   role='button'
                   aria-haspopup='true'
+                  ariaLabel='More items'
```

Comment:
>'More items' [](start = 28, length = 12)

we should add a prop to the Breadcrumb types for this aria label to be localized and customizable by the consuming. And the english text could be the default backup.

Reply:
We put no default localized strings anywhere in Fabric.

Reply:
(And if they're around, they're a bug.

Reply:
I added an optional prop for the overflow aria label.